tests/vision/checks/model_evaluation/train_test_prediction_drift_test.py

```diff
@@ -250,33 +250,3 @@ def test_condition_pass(mnist_dataset_train, mock_trained_mnist, device):
     ))
 
 
-def test_with_drift_object_detection_defected_alternative_measurements():
```

Comment:
Why delete these tests?

Reply:
@noamzbr the validation moved to the batch_wrapper so the tests moved to properties_utils_test

Reply:
But I don't see that properties_utils_test tests things like alternative measurements

Reply:
test_validate_properties_with_incorrect_property_dict_structure
test_validate_properties_with_bad_name_field
test_validate_properties_with_incorrect_property_output_type

Reply:
actually, most of the tests there test different types of defected alternative measurements